test/client/data/code-instrumentation/iframe.html

```diff
@@ -15,7 +15,7 @@
         crossDomainProxyPort: 2000
     });
 
-    Hammerhead.ShadowUI.getRoot();
+    Hammerhead.shadowUI.getRoot();
```

Comment:
`Hammerhead` should be lowercased since it's not a ctor. But this issue will be auto-resolved of we'll fix #18 in context of this PR.